modules/secured-cluster-configuration-options-operator.adoc

```diff
@@ -63,7 +63,10 @@ The default value is `false`.
 | Use this parameter to bypass the admission controller.
 
 | `admissionControl.contactImageScanners`
-| Specify `true` to enable inline scanning of images that are not already scanned during a deployment's admission review.
+a| Use one of the following values to specify when should the admission controller connect to the image scanner:
```

Comment:
```suggestion
| Use one of the following values to specify when should the admission controller connect to the image scanner:
```

Reply:
Thanks @porridge In AsciiDoc, the `a` before the cell is to allow bullet list inside the table cell.

Reply:
TIL 🤦🏻